### PR TITLE
docs(core): small fixes

### PR DESCRIPTION
**Pull request by isaacplmann**
Lots of small fixes:
- Moves convert from standalone to integrated repo next to the other convert repo recipes in the sidebar
- Formatting in Add Nuxt recipe
- Fix typo that hides entire Add Solid recipe
- Fix rounded corners when rescope message is under shell component

### CODE REVIEW

File: docs/shared/recipes/add-stack/add-nuxt.md [hunk 1]
```diff
@@ -24,8 +24,7 @@ Because we are not using an Nx plugin for Nuxt, there are few items we'll have t
 
 Using `nuxi init`, `cd` to the directory above where you want the app folder to live and run
 
-```shell
-❯ npx nuxi@latest init <app-name>
+```{% command="npx nuxi@latest init <app-name>" %}
```
Comment (juristr) [inline docs/shared/recipes/add-stack/add-nuxt.md:27]: why was it important to show the output here?

**Review by juristr — APPROVED** [review]

File: docs/shared/recipes/add-stack/add-solid.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 The code for this example is available on GitHub:
 
-{% github-repository url="https://github.com/nrwl/nx-recipes/tree/main/solidjs" %}
+{% github-repository url="https://github.com/nrwl/nx-recipes/tree/main/solidjs" /%}
```
Comment (meeroslav) [inline docs/shared/recipes/add-stack/add-solid.md:5]: Why do we have the backslash only here? Why is this needed?

**Review by meeroslav — CHANGES_REQUESTED** [review]
Some minor changes needed

File: docs/shared/recipes/add-stack/add-solid.md (hunk 2 above)
Comment (isaacplmann) [reply docs/shared/recipes/add-stack/add-solid.md:5]: If the backslash isn't there, the entire rest of the article is hidden.  I'm not sure the technical reason why that happens.

**Review by meeroslav — APPROVED** [review]
